parser/charset/encoding_table.go

```diff
@@ -298,3 +300,174 @@ func characterLengthUTF8(bs []byte) int {
 	}
 	return 4
 }
+
+func characterLengthOne(_ []byte) int {
+	return 1
+}
+
+type TruncateStrategy int8
+
+const (
+	TruncateStrategyEmpty TruncateStrategy = iota
+	TruncateStrategyTrim
+	TruncateStrategyReplace
```

Comment:
Please add some comments for these strategies.

Reply:
Updated.